apis/ops/v1alpha1/type.go

```diff
@@ -18,35 +18,55 @@ package v1alpha1
 
 // List of possible condition types for a ops request
 const (
-	AccessApproved            = "Approved"
-	AccessDenied              = "Denied"
-	DisableSharding           = "DisableSharding"
-	EnableSharding            = "EnableSharding"
-	Failure                   = "Failure"
-	HorizontalScalingDatabase = "HorizontalScaling"
-	MigratingData             = "MigratingData"
-	NodeCreated               = "NodeCreated"
-	NodeDeleted               = "NodeDeleted"
-	NodeRestarted             = "NodeRestarted"
-	PauseDatabase             = "PauseDatabase"
-	PausingDatabase           = "PausingDatabase"
-	PausedDatabase            = "PausedDatabase"
-	Progressing               = "Progressing"
-	ResumeDatabase            = "ResumeDatabase"
-	ResumingDatabase          = "ResumingDatabase"
-	ResumedDatabase           = "ResumedDatabase"
-	ScalingDatabase           = "Scaling"
-	ScalingDown               = "ScalingDown"
-	ScalingUp                 = "ScalingUp"
-	StartingBalancer          = "StartingBalancer"
-	StoppingBalancer          = "StoppingBalancer"
-	Successful                = "Successful"
-	Updating                  = "Updating"
-	UpgradedVersion           = "UpgradedVersion"
-	UpgradingVersion          = "UpgradingVersion"
-	VerticalScalingDatabase   = "VerticalScaling"
-	VotingExclusionAdded      = "VotingExclusionAdded"
-	VotingExclusionDeleted    = "VotingExclusionDeleted"
+	AccessApproved              = "Approved"
+	AccessDenied                = "Denied"
+	DisableSharding             = "DisableSharding"
+	EnableSharding              = "EnableSharding"
+	Failure                     = "Failure"
+	HorizontalScalingDatabase   = "HorizontalScaling"
+	MigratingData               = "MigratingData"
+	NodeCreated                 = "NodeCreated"
+	NodeDeleted                 = "NodeDeleted"
+	NodeRestarted               = "NodeRestarted"
+	PauseDatabase               = "PauseDatabase"
+	PausingDatabase             = "PausingDatabase"
+	PausedDatabase              = "PausedDatabase"
+	Progressing                 = "Progressing"
+	ResumeDatabase              = "ResumeDatabase"
+	ResumingDatabase            = "ResumingDatabase"
+	ResumedDatabase             = "ResumedDatabase"
+	ScalingDatabase             = "Scaling"
+	ScalingDown                 = "ScalingDown"
+	ScalingUp                   = "ScalingUp"
+	StartingBalancer            = "StartingBalancer"
+	StoppingBalancer            = "StoppingBalancer"
+	Successful                  = "Successful"
+	Updating                    = "Updating"
+	UpgradedVersion             = "UpgradedVersion"
+	UpgradingVersion            = "UpgradingVersion"
+	VerticalScalingDatabase     = "VerticalScaling"
+	VotingExclusionAdded        = "VotingExclusionAdded"
+	VotingExclusionDeleted      = "VotingExclusionDeleted"
+	UpdateStatefulSets          = "UpdateStatefulSets"
```

Comment:
Should we add a comment block to show Mongo specific ones? May be add `Mongo***` to make it clear in the name?